.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+      time: "10:00"
+    open-pull-requests-limit: 100
```

Comment:
What about limiting the number to a max of 10 so we don't have many "pending" PR's. Any thoughts about it?

Reply:
Agree, fixed